adapter/internal/mgw/mgw.go

```diff
@@ -185,6 +185,14 @@ func Run(conf *config.Config) {
 	// Set enforcer startup configs
 	xds.UpdateEnforcerConfig(conf)
 
+	enableJwtIssuer := conf.Enforcer.JwtIssuer.Enabled
+	if enableJwtIssuer {
+		label := "Production and Sandbox"
```

Comment:
For now label should be read by env or config

Reply:
yes, will do the changes. We can read this controlPlane.EnvironmentLabels config

Reply:
Fixed